NuGet.config

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
+    <add key="dotnetcore" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
A very recent update here https://github.com/dotnet/standard/issues/295#issuecomment-293930022, so yeah we need to update the projects in mirror to enable us to bring in this package.